posthog/settings.py

```diff
@@ -68,6 +69,12 @@ def print_warning(warning_lines: Sequence[str]):
 else:
     JS_URL = os.environ.get("JS_URL", "")
 
+PLUGINS_ENABLED = get_bool_from_env("PLUGINS_ENABLED", False)
+PLUGINS_INSTALL_FROM_WEB = PLUGINS_ENABLED and get_bool_from_env("PLUGINS_INSTALL_FROM_WEB", True)
+PLUGINS_CONFIGURE_FROM_WEB = PLUGINS_INSTALL_FROM_WEB or get_bool_from_env("PLUGINS_CONFIGURE_FROM_WEB", True)
```

Comment:
Should these perhaps default to False?

Advantage: Less surprises security-wise for users
Disadvantage: Update will not be smooth

Reply:
I would like to get this PR with these ENVs merged and then revisit this, hook it up to the organization system and then get rid of the ENVs. E.g. instance admins should be able to install plugins, etc. Though it looks like now this functionality is not yet fully there. I can't change the permissions and they don't seem to have much effect anyway.

![image](https://user-images.githubusercontent.com/53387/97708001-61ad8b00-1ab8-11eb-8113-e281ec39e96c.png)

Reply:
Sounds fair, thanks!